tools/param-name-importer/param-name-importer.csproj

```diff
@@ -9,10 +9,16 @@
   <ItemGroup>
     <!-- Not sure why needed, but only System.IO.Compression.FileSystem.dll is
       included by default, and ZipArchive is type forwarded to System.IO.Compression -->
-    <Reference Include="System.IO.Compression" />
+    <Reference Include="System.IO.Compression" Condition="'$(TargetFramework)' == 'net472'" />
```

Comment:
I half wonder if this shouldn't instead be `$(TargetFramework.StartsWith('net4'))`, to "reasonably" support .NET Framework 4.8…

Reply:
Fixed!